### PR TITLE
docs(fdp): update example FDP notebook to conform to default shapes

### PR DESCRIPTION
**Pull request by Markus92**
This PR updates the FDP example notebook for the default shapes.

Also fixed a few spelling and grammatical errors while at it.

### CODE REVIEW

File: docs/Usage_example_FDP.ipynb [hunk 1]
```diff
@@ -299,7 +125,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "catalog.publisher = [\"https://harrypotter.fandom.com/wiki/Hogwarts_School_of_Witchcraft_and_Wizardry\"]\n",
     "catalog_record = catalog.to_graph(catalog_subject)\n",
     "print(catalog_record.serialize())"
```
Comment (hcvdwerf) [inline docs/Usage_example_FDP.ipynb:129]: Is deze extra print noodzakelijk ?

File: docs/Usage_example_FDP.ipynb [hunk 2]
```diff
@@ -474,14 +342,25 @@
     "from sempyro.dcat import DCATDataset\n",
     "\n",
     "class FDPDataset(DCATDataset):\n",
```
Comment (hcvdwerf) [inline docs/Usage_example_FDP.ipynb:344]: HRIDataset zou afdoende moeten zijn

File: docs/Usage_example_FDP.ipynb [hunk 3]
```diff
@@ -505,17 +384,21 @@
...
-    "        keyword=[LiteralField(value=x) for x in record[\"keywords\"]]\n",
+    "        keyword=[LiteralField(value=x) for x in record[\"keywords\"]],\n",
+    "        has_version=\"0.1\",\n",
     "    )\n",
     "    dataset_subject = URIRef(f\"http://example.com/dataset_{record['id'][0]}\")\n",
     "    dataset_graph = dataset.to_graph(dataset_subject)\n",
     "    print(dataset_graph.serialize())\n",
-    "    fdp_client.create_and_publish(resource_type=\"dataset\", metadata=dataset_graph)\n"
+    "    dataset_fdp_id = fdpclient.create(type=\"dataset\", data=dataset_graph)\n",
+    "    fdpclient.update(\n",
```
Comment (hcvdwerf) [inline docs/Usage_example_FDP.ipynb:399]: RuntimeError                              Traceback (most recent call last)
Cell In[27], line 18
     16 dataset_graph = dataset.to_graph(dataset_subject)
     17 print(dataset_graph.serialize())
---> 18 dataset_fdp_id = fdpclient.create(type="dataset", data=dataset_graph)
     19 fdpclient.update(
     20     type="dataset", subtype="meta/state", id=dataset_fdp_id, data='{"current": "PUBLISHED"}', format="json-ld"
     21 )

File ~/Github/SeMPyRO/venv/lib/python3.9/site-packages/fdpAPIconnector/fdpclient.py:98, in FDPClient.create(self, type, data, format)
     96 except Exception as error:
     97     print(f'Unexpected error when connecting to {self.baseurl}/{type}\n')
---> 98     raise error

File ~/Github/SeMPyRO/venv/lib/python3.9/site-packages/fdpAPIconnector/fdpclient.py:94, in FDPClient.create(self, type, data, format)
     91     if r.status_code >= 300:
     92         print(f'HTTP error: {r.status_code} {r.reason} for {self.baseurl}/{type}',
     93               f'\nResponse message: {r.text}')
---> 94         raise
     95     return self._removePrefix(r.headers['Location'],f'{self.publicurl}/{type}/')
     96 except Exception as error:

RuntimeError: No active exception to reraise

Comment (hcvdwerf) [reply docs/Usage_example_FDP.ipynb:399]: Bij mij gaat het niet helemaal lekker

**Review by hcvdwerf — CHANGES_REQUESTED** [review]

File: docs/Usage_example_FDP.ipynb (hunk 2 above)
Comment (Markus92) [reply docs/Usage_example_FDP.ipynb:344]: HRIDataset is anders dan default shapes op v1.16 FDP

**Pull request merged**